data/alpha/register/registration-district.yaml

```diff
@@ -7,4 +7,4 @@ fields:
 phase: alpha
 register: registration-district
 registry: hm-passport-office
-text: Registration districts in England and Wales, defined for the purposes of the Registration Acts
+text: Registration districts in England and Wales, defined for the purposes of the Civil Registrations Acts
```

Comment:
This is not a you issue, we have not been doing this well traditionally and I would like to improve it.